scripts/devnet-curio/curio.dockerfile

```diff
@@ -0,0 +1,103 @@
+#####################################
+ARG LOTUS_IMAGE
+# hadolint ignore=DL3006
+FROM ${LOTUS_IMAGE} as lotus-test
+FROM golang:1.22.3-bullseye AS curio-builder
```

Comment:
How about using `debian:bookworm` and `ubuntu:noble`

Reply:
I'd like to keep it as similar to the "official" curio build environment as possible.